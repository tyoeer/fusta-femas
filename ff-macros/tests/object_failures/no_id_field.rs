use ff_macros::*;

#[derive(Object)]
struct Test {
	not_id: i32,
	also_not_id: bool,
}

fn main() {}