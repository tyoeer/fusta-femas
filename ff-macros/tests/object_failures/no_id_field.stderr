error: no id field
 --> tests/object_failures/no_id_field.rs:5:2
  |
5 | /     not_id: i32,
6 | |     also_not_id: bool,
  | |______________________^
